### DISCUSSION TITLE
Scheduler memory usage since v2.9.0

### DISCUSSION THREAD

**Discussion by sseelmann** [apache/airflow#39202] [General]
Hello, I wanted to ask if it's a know issue or fact that the scheduler requires more memory since version 2.9.0? We run Airflow in EKS using the Helm chart, and since update to version 2.9.0 we see a significant increase in memory, see attached metrics screenshot.

![Screenshot_2024-04-23_13-30-26](https://github.com/apache/airflow/assets/72128241/3d2e65de-6797-4687-9b7a-2d1ddca6b1b7)

Kind regards,
Stefan

**Comment by matteosdocsity** [comment]
I'm experiencing the same behaviour for both LocalExecutor and SequentialExecutor

**Comment by f-hluchnik** [comment]
Hi, I'm experiencing the same issue too. Have you found some solution to your problem?

**Reply by sseelmann** [reply]
We just increased the memory from 1GB to 1.5GB to make it work :shrug:
